docs/eventing/sources/sources.yaml

```diff
@@ -24,6 +24,14 @@ metaSources:
     description: >
       SampleSource is a reference implementation supporting the
       [Writing an Event Source the Hard Way tutorial](../samples/writing-a-source).
+  - name: Sample Source
```

Comment:
I think it would be helpful to give this source a unique name. Maybe "Sample Source with Receive Adapter"?

Reply:
So this might be worth having a discussion over, but the other sample source doesn't seem to have a receive adapter and seems closer to the sample-controller repo? and that it's built with kubebuilder? @grantr any suggestions on the naming collision?

Reply:
Feel free to rename the existing Sample Source. It's incomplete (because it doesn't have a receive adapter) and probably obsolete. Plus it uses kubebuilder. @lberk, I suspect yours is better :)

Reply:
After checking with @grantr I've changed the references to `Kubebuilder SampleSource` and left the tutorial in this PR as-is in f1e2ac5